Makefile

```diff
@@ -1,6 +1,6 @@
 
 # Image URL to use all building/pushing image targets
-IMG ?= controller:latest
+IMG ?= quay.io/konveyor/oadp-non-admin:latest
```

Comment:
We currently have:

 https://quay.io/repository/migtools/oadp-nac
 
 Should we use the above or create new one within migtools and not konveyor?

Reply:
we configured the above one in PROW https://github.com/openshift/release/blob/master/core-services/image-mirroring/konveyor/mapping_konveyor_latest#L41